targets/targets.json

```diff
@@ -1724,6 +1724,40 @@
         "release_versions": ["2"],
         "device_name": "STM32F407VG"
     },
+    "WIO_3G": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M4F",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSE_XTAL|USE_PLL_HSI",
```

Comment:
Question: What is the HW patch? Is this explained somewhere in the docs?

Reply:
Oops, no need hardware patch. XTAL is mounted on the board. Will remove "need HW patch" on next commit.

Reply:
Done